ur_controllers/src/gpio_controller.cpp

```diff
@@ -100,6 +100,10 @@ controller_interface::InterfaceConfiguration ur_controllers::GPIOController::sta
   for (size_t i = 0; i < 11; ++i) {
     config.names.emplace_back("gpio/safety_status_bit_" + std::to_string(i));
   }
+  config.names.emplace_back("system_interface/initialized");
```

Comment:
That would probably require adding this to the xacro, as well. (For Foxy we decided to stick with the one in this repo)

Reply:
resolved in [7ea0d27](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/324/commits/7ea0d27263aa4bcba4f059a3db4582e1221ffe71)